server/src/main/java/io/crate/execution/engine/aggregation/impl/CountAggregation.java

```diff
@@ -276,4 +283,31 @@ public MutableLong removeFromAggregatedState(RamAccounting ramAccounting,
         }
         return null;
     }
+
+    @Nullable
+    @Override
+    public DocValueAggregator<?> getDocValueAggregator(List<Symbol> aggregationReferences,
+                                                       java.util.function.Function<List<String>, List<MappedFieldType>> getMappedFieldTypes,
+                                                       DocTableInfo table,
+                                                       List<Literal<?>> optionalParams) {
+        if (aggregationReferences.size() == 1 && aggregationReferences.get(0).valueType().id() == ObjectType.ID && table != null) {
```

Comment:
In which cases can `table` be null? It is also not marked as `Nullable` ?

Reply:
table will not be null since it went through the analyzer, removing